codecov.yml

```diff
@@ -7,3 +7,5 @@ coverage:
       python:
         target: 90%
 comment: false
+ignore:
+  - "tests"
```

Comment:
I sneaked "Codecov, please ignore the `tests/` directory in here, because I am too lazy to create a new PR to do this two lines.